### PR TITLE
Acceptance test: breadcrumb is not seen one level below root directory when rootFolder is set

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

When ```rootFolder``` is set in ```dist/config.json```, then breadcrumb is not seen one level below the root directory. For more info check this issue: https://github.com/owncloud/phoenix/issues/2538
## Related Issue

https://github.com/owncloud/phoenix/issues/2538

## How Has This Been Tested?

- CI
- Manual
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/stepDefinitions/generalContext.js [hunk 1]
```diff
@@ -162,7 +192,7 @@ After(async function () {
   afterStdOut = JSON.parse(afterStdOut)
   const appConfigSettings = _.get(afterStdOut, 'apps')
   if (appConfigSettings === undefined) {
-    throw new Error("'apps' was not found inside stdOut of response.")
+    throw new Error('\'apps\' was not found inside stdOut of response.')
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/generalContext.js]: Please revert.

File: tests/acceptance/features/webUIFiles/breadcrumb.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  As a user
+  I want to browse to parent folders using breadcrumb
+  So that I can access resources with ease
+
+  Background:
+    Given user "user1" has been created with default attributes
+
+  @issue-2538
+  Scenario: Check breadCrumb for folder one level below the root folder when rootFolder is set
+    Given rootFolder has been set to "" in config.json
```
Comment (dpakach) [inline tests/acceptance/features/webUIFiles/breadcrumb.feature]:
```suggestion
    Given the rootFolder has been set to ""
```
I dont think it is a better idea to give file name in the feature file what do you think?

Comment (individual-it) [reply tests/acceptance/features/webUIFiles/breadcrumb.feature]: what about calling it "phoenix config file"?

File: tests/acceptance/features/webUIFiles/breadcrumb.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    Then breadcrumb for folder "simple-empty-folder" should be displayed on the webUI
+
+  Scenario: Change rootFolder to simple-folder and check for the displayed files
+    Given rootFolder has been set to "simple-folder" in config.json
+    And user "user1" has logged in using the webUI
+    When the user browses to the files page
+    Then folder "0" should not be listed on the webUI
+    But as "user1" folder "0" should exist
+    And file "lorem.txt" should be listed on the webUI
+    And the content of file "simple-folder/lorem.txt" for user "user1" with range "0-39" bytes should be "This is lorem text in the simple-folder"
```
Comment (individual-it) [inline tests/acceptance/features/webUIFiles/breadcrumb.feature]: I don't think we need that check here, the config change will only affect phoenix but that check is done by webDAV

**Comment by kiranparajuli589** [comment]
LGTM :+1:

**Pull request merged**